run/test/specs/disappearing_message_checks_android.spec.ts

```diff
@@ -0,0 +1,429 @@
+import { androidIt, iosIt } from "../../types/sessionIt";
+import { clickOnXAndYCoordinates, sleepFor } from "./utils";
+import { newUser } from "./utils/create_account";
+import { newContact } from "./utils/create_contact";
+import {
+  SupportedPlatformsType,
+  closeApp,
+  openAppTwoDevices,
+} from "./utils/open_app";
+import { setDisappearingMessage } from "./utils/set_disappearing_messages";
+import { runScriptAndLog } from "./utils/utilities";
+
+async function disappearingImageMessage(platform: SupportedPlatformsType) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  const testMessage = "Testing disappearing messages for images";
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
+  await newContact(platform, device1, userA, device2, userB);
+  await setDisappearingMessage(device1, [
+    "1o1",
+    "Disappear after send option",
+    "10 seconds",
+  ]);
+  await device1.navigateBack(platform);
+  await device1.sendImage(platform, testMessage);
+  await device2.clickOnElement("Untrusted attachment message");
+  await sleepFor(500);
+  // User B - Click on 'download'
+  await device2.clickOnElement("Download media");
+  await sleepFor(10000);
```

Comment:
same, would be good to not have an explicit sleepFor here, the image might take 1s or 20s to load